Tests/MessagingInApp/MessagingInAppImplementationTest.swift

```diff
@@ -184,33 +188,9 @@ class MessagingInAppImplementationTest: UnitTest {
         XCTAssertEqual(eventListenerMock.messageActionTakenReceivedArguments?.actionValue, givenAction)
         XCTAssertEqual(eventListenerMock.messageActionTakenReceivedArguments?.actionName, givenName)
 
+        // FIXME: [CDP] Test if the task is being forwarded to EventBus (use Mocks to test)
         // make sure there is no click tracking for "close" action
-        XCTAssertEqual(backgroundQueueMock.addTrackInAppDeliveryTaskCallsCount, 0)
-    }
-
-    func test_inAppTracking_givenCustomAction_expectBQTrackInAppClicked() {
```

Comment:
Why did we remove this?

Reply:
background queue is no longer used. As discussed earlier, keeping this method with a note to fix when eventBus is done.

Reply:
Got it